src/apify_helpers.js

```diff
@@ -213,6 +213,168 @@ const getPrefilledValuesFromInputSchema = (inputSchemaStringJSON) => {
+        {
+            label: actor.title,
+            key: `actor-${actor.id}-info`,
+            type: 'copy',
+            helpText: description || `${actor.title} Input, see [documentation](https://apify.com/${actor.username}/${actor.name}) `
+                + 'for detailed fields description.',
+        },
+    ];
+    // eslint-disable-next-line no-restricted-syntax
+    for (const [propertyKey, definition] of Object.entries(properties)) {
+        // eslint-disable-next-line no-continue
+        if (definition.editor === 'hidden') continue;
+        // NOTE: Handle sectionCaption with info box with helpText. It is not possible to do stackable fields in Zapier.
+        if (definition.sectionCaption && definition.sectionCaption.length) {
+            const helpText = definition.sectionDescription
+                ? `${definition.sectionCaption} - ${definition.sectionDescription}`
+                : definition.sectionCaption;
+            fields.push({
+                label: definition.sectionCaption,
+                key: `sectionCaption-${propertyKey}`,
+                type: 'copy',
+                helpText,
+            });
+        }
+        const field = {
+            label: definition.title,
+            helpText: definition.description,
+            key: propertyKey,
+            required: required && required.includes(propertyKey),
+            // NOTE: From Zapier docs: A default value that is saved the first time a Zap is created.
+            // It is what what prefill is in Apify input schema.
+            default: definition.prefill,
+            // NOTE: From Zapier docs: An example value that is not saved.
+            // It is what what default is in Apify input schema.
+            placeholder: definition.default,
+        };
+        switch (definition.type) {
+            case 'string': {
+                // NOTE: Cannot provide alternative in fields schema for options pattern, minLength, maxLength, nullable
+                // These options will not cover UI validation and we need to handle it in code.
+                field.type = 'string'; // editor = textfield, datepicker
+                if (['javascript', 'python'].includes(definition.editor)) {
+                    field.type = 'code';
+                } else if (definition.editor === 'textarea') {
+                    field.type = 'text';
+                } else if (definition.editor === 'datepicker') {
+                    field.type = 'datetime';
+                } else if (definition.editor === 'select') {
+                    field.choices = {};
+                    definition.enum.forEach((key, i) => {
+                        field.choices[key] = definition.enumTitles ? definition.enumTitles[i] : key;
+                    });
+                }
+                if (definition.isSecret) {
+                    field.type = 'password';
+                }
```

Comment:
The secret input is tricky. This way, I think it's going to be sent to our api as just text, right? Meaning it's going to be visible in input KVS etc. Maybe we should add UI to console that would allow to "encrypt any value to be used by this actor" and add this info to the field description in zapier?

Reply:
This is not the case if you send a secret in input we will encrypt it before storing it in kvs as input. 
Secret will be transferred only into the API server, but using HTTPS so it is fine.

Reply:
Ah, didn't know it's implemented like this, but it makes sense.